platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/files/DataFileWriter.java

```diff
@@ -62,6 +60,22 @@ public final class DataFileWriter {
      */
     private long dataItemCount = 0;
 
+    private final long dataBufferSize;
+
+    /**
+     * Create a new data file with moving mapped byte buffer of 256Mb size.
+     */
+    public DataFileWriter(
+            final String filePrefix,
+            final Path dataFileDir,
+            final int index,
+            final Instant creationTime,
+            final int compactionLevel)
+            throws IOException {
+        // TODO maybe we can use 128Mb buffer size (see DataFileWriterBenchmark)
```

Comment:
Let's not have any TODOs in the code. It should be either removed (with the best buffer size value used), or changed to FUTURE WORK + a link to a ticket used to track this work